packages/amplify-util-node-template-provider/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "amplify-util-node-template-provider",
```

Comment:
`amplify-nodejs-functon-template-provider` as we discussed

Reply:
fixed